cluster/test/Main.hs

```diff
@@ -45,6 +49,10 @@ main = evalResults
 -- fails if one or more them returned a failure.
 evalResults :: [IO (String, Result)] -> IO ()
 evalResults xs = do
+#if mingw32_HOST_OS
```

Comment:
Can we unconditionally set utf8 encoding and avoid CPP?

Reply:
This is mingw32 guarded for consistency. Same solution is in hedgehog, and a few other places.

Inspector has a slightly different fix: https://github.com/primetype/inspector/commit/c975f4329365f0379c04358138e616fb96fb0b79

This however only works if `Basement` is already part of the dependency tree.